packages/canner-wrapper/src/components/types.js

```diff
@@ -45,13 +45,20 @@ export type NavbarConfig = {
   showSaveButton: boolean
 }
 
+export type Router = {
+  getRoutes: () => Array<string>,
+  getParams: () => Object,
+  goTo: ({pathname: string, params: Object}) => void
+};
+
```

Comment:
@abz53378 @chihsuan the public methods in `Router` should be more specific.

Reply:
OK! So here is the new interface of `router`, how do you think?
```js
{
  getRoutes: () => Array<string>,
  getOperator: () => 'create' | 'update', // default update 
  getPayload: () => Object, // default empty object
  getWhere: () => Object, // default empty object
  getSort: () => Object, // default empty object 
  getPagination: () => Object // default empty object,
  goTo: ({
     pathname: string,
     operator?: 'create' | 'update',
     payload?: Object,
     where?: Object,
     sort?: Object,
     pagination?: Object
  }) => void
}
```

Also `<Canner>` will receive `operator`, `payload`, `where`, `sort`, `pagination` instead of `params`

Reply:
@abz53378 use `routerParams` instead to be more clear.

Reply:
@chihsuan, after the discussion, here are some changes:

- remove `getParams`, use `getOperator`, `getPayload`, `getWhere`, `getSort`, `getPagination` I mentioned above.
- change `goTo` API to the interface above.
- `@canner/container` pass `routerParams` to children instead of `params`.
  ```
  type routerParams = {
    operator?: 'create' | 'update',
    payload?: Object,
    where?: Object,
    sort?: Object,
    pagination?: Object
  }
  ```

- change the name of canner-wrapper to `@canner/container`(includes package name, folder name, class name)

Thanks!

Reply:
@abz53378: I've updated. Please review the changes. Thanks!